src/plugins/router.ts

```diff
@@ -59,103 +65,108 @@ const routes: RouteRecordRaw[] = [
     component: HomePage,
   },
   {
-    path: '/trade/:assetIn?/:assetOut?',
+    path: '/terms-of-use',
+    name: 'terms-of-use',
+    component: TermsOfUsePage,
+    meta: { layout: 'ContentLayout' },
+  },
+  {
+    path: '/privacy-policy',
+    name: 'privacy-policy',
+    component: PrivacyPolicyPage,
+    meta: { layout: 'ContentLayout' },
+  },
+  {
+    path: '/cookies-policy',
+    name: 'cookies-policy',
+    component: CookiesPolicyPage,
+    meta: { layout: 'ContentLayout' },
+  },
+  {
+    path: '/:networkSlug/trade/:assetIn?/:assetOut?',
     name: 'trade',
     component: TradePage,
   },
   {
-    path: '/swap/:assetIn?/:assetOut?',
+    path: '/:networkSlug/swap/:assetIn?/:assetOut?',
     redirect: to => {
       return `/trade${to.path.split('/swap')[1]}`;
     },
   },
   {
-    path: '/pool/create/:tx?',
+    path: '/:networkSlug/pool/create/:tx?',
     name: 'create-pool',
     component: CreatePoolPage,
     meta: { layout: 'FocusedLayout' },
   },
   {
-    path: '/pool/:id',
+    path: '/:networkSlug/pool/:id',
     name: 'pool',
     component: PoolPage,
   },
   {
-    path: '/pool/:id/invest',
+    path: '/:networkSlug/pool/:id/invest',
     name: 'invest',
     component: PoolInvestPage,
     meta: { layout: 'PoolTransferLayout' },
   },
   {
-    path: '/pool/:id/withdraw',
+    path: '/:networkSlug/pool/:id/withdraw',
     name: 'withdraw',
     component: PoolWithdrawPage,
     meta: { layout: 'PoolTransferLayout' },
   },
   {
-    path: '/pool/migrate/:from/:to',
+    path: '/:networkSlug/pool/migrate/:from/:to',
     name: 'migrate-pool',
     component: MigratePoolPage,
     meta: { layout: 'FocusedLayout' },
   },
   {
-    path: '/terms-of-use',
-    name: 'terms-of-use',
-    component: TermsOfUsePage,
-    meta: { layout: 'ContentLayout' },
-  },
-  {
-    path: '/privacy-policy',
-    name: 'privacy-policy',
-    component: PrivacyPolicyPage,
-    meta: { layout: 'ContentLayout' },
-  },
-  {
-    path: '/cookies-policy',
-    name: 'cookies-policy',
-    component: CookiesPolicyPage,
-    meta: { layout: 'ContentLayout' },
-  },
-  {
-    path: '/:pathMatch(.*)*',
-    name: 'not-found',
-    redirect: '/',
-  },
-  {
-    path: '/vebal',
+    path: '/:networkSlug/vebal',
```

Comment:
This page is only on mainnet, same with `get-vebal` and `unlock`

Reply:
Yes, I was thinking about this - I see what you mean. However, would it not be easier for users to understand this via the existing modal which explains that it's eth only? Might be confusing that all other urls start with the network name. 

Also, this is more future-proof if there is veBal on other networks.

Reply:
Yeah, I think it's fine to maintain the network in the URL for the verbal pages for the same points that Matthew made.